section2/exercises/if-statements.js

```diff
@@ -86,11 +93,19 @@ var cupsOfFlour = 1;
 var hasSauce = true;
 
 // Write your conditional here
-
-
+if (cupsOfFlour && hasSauce); {
```

Comment:
This is exactly working as we want it to right now.

You'll need to check something against `cupsOfFlour`. Go back to the prompt, what _about_ `cupsOfFlour` matters?

Also, if we want line 98 to be checked after line 96, it should be an `else if`. 

I'd recommend writing out in plain English how you'd solve/check for these things, then translating it bit by bit into JavaScript - then make sure to run the code to verify it works! If it's working now, we have a false positive.